config/routes.rb

```diff
@@ -884,7 +884,7 @@
       as: 'comments'
 
   get 'main/sponsors',
-      to: 'main#sponsors',
+      to: redirect('#sponsors'),
       as: 'sponsors'
```

Comment:
Why still keep the 'main/sponsors' route instead of deleting it?

Reply:
I thought of external websites that may have linked to the sponsors website and now get a broken link... But if that doesn't matter, we can of course get rid of it.

Reply:
I would say we get rid of it.